app/renderer/components/preferences/payment/bitcoinDashboard.js

```diff
@@ -341,7 +341,7 @@ class BitcoinDashboard extends ImmutableComponent {
                         {/* TODO: refactor button.js */}
                         <Button className={cx({
                           primaryButton: true,
-                          [css(globalStyles.spacing.dialogInsideMargin)]: true
+                          [css(styles.bitcoinAddressButton)]: true
```

Comment:
`dialogInsideMarginTop` and `dialogInsideMarginBottom`.

As this is not a big change, I'm not strongly against bitcoinAddressButton.